<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-OEJR8N7" name="com.TestNg.Expected_Exception" tests="3" failures="0" timestamp="2022-02-22T14:36:31 IST" time="0.045" errors="0">
  <testcase name="nullPointer_Excep" time="0.000" classname="com.TestNg.Expected_Exception"/>
  <system-out/>
  <testcase name="stringIndex_OutofBounds" time="0.016" classname="com.TestNg.Expected_Exception"/>
  <system-out/>
  <testcase name="arithmetic_Excep" time="0.029" classname="com.TestNg.Expected_Exception"/>
  <system-out/>
</testsuite> <!-- com.TestNg.Expected_Exception -->
